packages/ui-bindings/EconomicEvent/old_code

```diff
@@ -0,0 +1,87 @@
+
```

Comment:
If this file is available in a different commit, I'd suggest deleting it since it's just old code

Reply:
^what he said

Reply:
Deleted